README.md

```diff
@@ -1 +1,23 @@
 # Deployer
+This is a proof of concept that will deploy a VM in UCL RSDG's Azure
+subscription given a series of parameters and a Dockerfile entered on a
+web form .
+
+In order to run this PoC you'll need to:
+
+1. Clone this repo.
+1. Make sure you have Python3 installed in your system.
+1. Install project's requirements:
+    `pip install -r requirements`
+1. You'll need `variables.tf`, a Terraform file containing the values of the cloud provider secrets needed to access your subscription. See [Terraform's sample for AWS](https://www.terraform.io/intro/getting-started/variables.html#using-variables-in-configuration).
```

Comment:
In Dashboard we've committed a template file with placeholders to fill in; might be worth doing that here? Then devs can just copy & edit the template.